meta-security-smack/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -15,6 +15,21 @@ _ System -wx---
 ^ System::Run rwxat-
 ^ System rwxa--
 EOF
+
+    mkdir -p ${D}/${libdir}/tmpfiles.d
```

Comment:
Please use "install -d" instead. That takes care of setting permissions, something that mkdir alone does not do reliably.